docs/source/ops/connect/auth0.rst

```diff
@@ -0,0 +1,657 @@
+your app folder (``my-project`` in this example):
+
+.. code-block:: bash
+
+    #!/usr/bin/env bash
+
+    set -euo pipefail
+    openssl req -x509 \
+                -newkey rsa:4096 \
+                -keyout /etc/ssl/private/nginx-selfsigned.key \
+                -out /etc/ssl/certs/nginx-selfsigned.crt \
+                -days 365 \
+                -nodes \
+                -subj "/C=US/ST=Oregon/L=Portland/O=Company Name/OU=Org/CN=${FRONTEND_IP}"
+    openssl dhparam -out /etc/ssl/certs/dhparam.pem 2048
+    cat <<NGINX_CONFIG > /etc/nginx/nginx.conf
+    worker_processes auto;
+    pid /run/nginx.pid;
+    events {
+      worker_connections 768;
+    }
+    http {
+      sendfile on;
+      tcp_nopush on;
+      tcp_nodelay on;
+      keepalive_timeout 65;
+      types_hash_max_size 2048;
+      include /etc/nginx/mime.types;
+      default_type application/octet-stream;
+      access_log /var/log/nginx/access.log;
+      error_log /var/log/nginx/error.log;
+      gzip on;
+
+      ssl_certificate /etc/ssl/certs/nginx-selfsigned.crt;
+      ssl_certificate_key /etc/ssl/private/nginx-selfsigned.key;
+      ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+      ssl_prefer_server_ciphers on;
+      ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+      ssl_ecdh_curve secp384r1;
+      ssl_session_cache shared:SSL:10m;
+      ssl_session_tickets off;
+      ssl_stapling on;
+      ssl_stapling_verify on;
+      resolver 8.8.8.8 8.8.4.4 valid=300s;
+      resolver_timeout 5s;
+      add_header X-Frame-Options DENY;
+      add_header X-Content-Type-Options nosniff;
+
+      ssl_dhparam /etc/ssl/certs/dhparam.pem;
+
+      server {
+        listen 80;
+        return 302 https://${FRONTEND_IP}\$request_uri;
+      }
+
+      server {
+        listen 443 ssl http2;
+        location /v1/stream {
+          proxy_pass http://${JSON_IP};
+          proxy_http_version 1.1;
+          proxy_set_header Upgrade \$http_upgrade;
+          proxy_set_header Connection "Upgrade";
+          proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+        }
+        location /v1 {
+          proxy_pass http://${JSON_IP};
+          proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+        }
+        location /auth/ {
+          proxy_pass http://${AUTH_IP}/;
+          proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+        }
+        location /trigger/ {
+          proxy_pass http://${TRIGGER_IP}/;
+          proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+        }
+        root /app/ui;
+        index index.html;
+        location / {
+          # for development, uncomment proxy_pass and comment the try_files line
+          #proxy_pass http://localhost:3000/;
+          try_files \$uri \$uri/ =404;
+        }
+      }
+    }
+    NGINX_CONFIG
+
+Note: if you want to work on your frontend code while otherwise using the rest
```

Comment:
did you intend for this to be a  `.. note:: `

Reply:
Ah, yes, I was not aware of that directive. Thanks!